source/installation/00-overview.md

```diff
@@ -1,9 +1,12 @@
 # Overview
 
+The NodeX agent is currently available as a pre-built binary download from the release page in the GitHub repository. In the future, we are working on making it easier for developers to install the agent by using the distribution's official package management system to provide a better installation and development experience.
+
+Future plans include providing pre-built binaries for WindowsOS. In addition, we plan to provide C/C++/Rust SDK for LinuxOS/WindowsOS/macOS and C/C++ SDK for RTOS (FreeRTOS, AzureRTOS).
```

Comment:
As of now, it has been verified to work on Ubuntu, Debian, Yocto, and macOS. Since this document describes the installation procedures, it would be beneficial to include the required version information (e.g., "The NodeX Agent v2.0 requires macOS xx.xx or higher.").